frontend/src/components/activity/ButtonNestedContentNodeAdd.vue

```diff
@@ -95,7 +96,7 @@ export default {
 
         await this.allContentNodes().$reload()
       } catch (error) {
-        console.log(error) // TO DO: display error message in error snackbar/toast
+        this.$toast.error(multiLineToast(violationsToFlatArray(error)))
```

Comment:
Consider exporting a single function from toasts.js which combines these two functions. You could call it e.g. `errorToMultilineToast`

Reply:
done